### PR TITLE
use ubi8/ubi-minimal base image instead of centos

### PR DESCRIPTION
**Pull request by ksimon1**
cc @fromanirh, @MarSik, @yanirq

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,11 +1,10 @@
-FROM centos:7
+FROM registry.access.redhat.com/ubi8/ubi-minimal
 
 LABEL maintainer="ksimon@redhat.com"
 
 ENV container docker
 
-RUN yum -y update
-RUN yum clean all
+RUN microdnf -y update && microdnf clean all
```
Comment (fromanirh) [inline Dockerfile]: not sure this is needed, I'm pruning this step from my images for the time being. I think you can keep it if you wish.

File: Dockerfile [hunk 2]
```diff
@@ -1,12 +1,9 @@
-FROM centos:7
+FROM registry.access.redhat.com/ubi8/ubi-minimal
```
Comment (MarSik) [inline Dockerfile]: We should use the 8-released tag if available, to avoid pre-releases

Comment (fromanirh) [reply Dockerfile]: Looks like the only tags available are "8.0" and "latest"
https://access.redhat.com/containers/?tab=tags#/registry.access.redhat.com/ubi8/ubi

Comment (ksimon1) [reply Dockerfile]: I think, this tag is not available for this image.
`Step 1/5 : FROM registry.access.redhat.com/ubi8/ubi-minimal:8-released
error parsing HTTP 404 response body: invalid character 'F' looking for beginning of value: "File not found.\""
`

**Pull request merged**